geode-wan/src/test/java/org/apache/geode/internal/cache/wan/wancommand/WanCommandCreateDestroyGatewaySenderDUnitTest.java

```diff
@@ -55,7 +55,7 @@ private CommandResult executeCommandWithIgnoredExceptions(String command) {
    * GatewaySender with all default attributes
```

Comment:
Would it make sense to create different tests for destroy and keep them separate from testing create?  That way when a test fails we know exactly what is failing (create vs destroy).